src/lib/types/order.ts

```diff
@@ -0,0 +1,17 @@
+
+// Add new type for orders
+export interface EngineOrder {
+    orderId: string;
+    clientOrderId: string;
```

Comment:
```suggestion
    customOrderId: string;
```

More intuitive like this, since it doesn't need to be tied to what binance calls it.